modules/network/src/integrationTest/java/org/apache/ignite/internal/network/netty/ItConnectionManagerTest.java

```diff
@@ -87,6 +94,11 @@ final void tearDown() throws Exception {
         IgniteUtils.closeAll(startedManagers);
     }
 
+    @BeforeAll
+    static void registerChannel() {
+        testChannel = ChannelInfo.generate("ItConnectionManagerTest");
```

Comment:
I liked the name `ChannelType` more. Right now I can't understand, what type of "info" it holds